documentation-site/components/header-navigation.js

```diff
@@ -0,0 +1,156 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+import Link from 'next/link';
+import {Block} from 'baseui/block';
+import {Tag} from 'baseui/tag';
+import {
+  HeaderNavigation,
+  StyledNavigationList as NavigationList,
+  ALIGN,
+} from 'baseui/header-navigation';
+import {styled} from 'baseui';
+import Menu from 'baseui/icon/menu';
+import Logo from '../images/base-web.svg';
+import GithubLogo from './github-logo';
+import SlackLogo from './slack-logo';
+import Search from './search';
+import {version} from '../../package.json';
+
+export const HEADER_BREAKPOINT = '@media screen and (min-width: 640px)';
+
+const Hamburger = styled('div', ({$theme}) => ({
+  display: 'block',
+  userSelect: 'none',
+  paddingLeft: $theme.sizing.scale600,
+  paddingTop: '2px',
```

Comment:
```
paddingTop: $theme.sizing.scale0,
```

Reply:
I personally like using github's suggest feature for single line changes https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-a-suggested-change

```suggestion
  paddingTop: $theme.sizing.scale0,
```

Reply:
Ohh didn't realize that icon that did that. Amazing